gdx/test/com/badlogic/gdx/math/IntersectorTest.java

```diff
@@ -0,0 +1,65 @@
+
+package com.badlogic.gdx.math;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+import com.badlogic.gdx.math.*;
+import com.badlogic.gdx.math.collision.*;
+import com.badlogic.gdx.math.Intersector.*;
+
+public class IntersectorTest {
+
+	@Test
+	public void test_circle () {
+		Circle c1 = new Circle(0, 0, 1);
+		Circle c2 = new Circle(0, 0, 1);
+		Circle c3 = new Circle(2, 0, 1);
+		Circle c4 = new Circle(0, 0, 2);
+
+		assertTrue(c1.overlaps(c1));
+		assertTrue(c1.overlaps(c2));
+		assertFalse(c1.overlaps(c3));
+		assertTrue(c1.overlaps(c4));
+		assertTrue(c4.overlaps(c1));
+		assertTrue(c1.contains(0, 1));
+		assertFalse(c1.contains(0, 2));
+		assertTrue(c1.contains(c1));
+		assertFalse(c1.contains(c4));
+		assertTrue(c4.contains(c1));
+	}
+
+	@Test
+	public void test_rectangle () {
+		Rectangle r1 = new Rectangle(0, 0, 1, 1);
+		Rectangle r2 = new Rectangle(1, 0, 2, 1);
+		assertTrue(r1.overlaps(r1));
+		assertFalse(r1.overlaps(r2));
+		assertTrue(r1.contains(0, 0));
+	}
+
+	@Test
+	public void test_boundingBox () {
+		BoundingBox b1 = new BoundingBox(Vector3.Zero, new Vector3(1, 1, 1));
+		BoundingBox b2 = new BoundingBox(new Vector3(1, 1, 1), new Vector3(2, 2, 2));
+		assertTrue(b1.contains(Vector3.Zero));
+		assertTrue(b1.contains(b1));
+		assertFalse(b1.contains(b2));
+		// Note, in stage the bottom and left sides are inclusive while the right and top sides are exclusive.
+	}
+
+	//@Test //No reason to test without asserts
+	public void intersectorTest () {
+		Plane plane = new Plane(new Vector3(1, 0, 0), 0);
+		SplitTriangle split = new SplitTriangle(3);
+		float[] fTriangle = {-10, 0, 10, -1, 0, 0, -10, 0, 10};
+		Intersector.splitTriangle(fTriangle, plane, split);
+		System.out.println(split);
```

Comment:
I'd like to add proper asserts, but as I said before, I don't know what to assert for. I could assert that it outputs what it does now, but how can I know that it works now? If anybody knows what it should output, I can add it there.